commands-v2/ramsete/constants.py

```diff
@@ -0,0 +1,75 @@
+from commands2 import RamseteCommand
+
+from wpilib import AnalogGyro
+
+from wpimath.kinematics import DifferentialDriveKinematics
+
+from math import pi
+
+"""
+A place for the constant values in the code
+that may be used in more than one place. 
+"""
+
+
+class Constants:
```

Comment:
Java puts its constants in a class because things cannot exist outside of a class. There's no reason one needs to do that in python, having a constants module is sufficient. See the hatchbot example I added.

Reply:
Ok, will change that.